scripts/dispatch_table_helper_generator.py

```diff
@@ -226,7 +226,7 @@ def getTypeNameTuple(self, param):
     def OutputExtEnabledFunction(self):
         ext_fcn = ''
         # First, write out our static data structure -- map of all APIs that are part of extensions to their extension.
-        ext_fcn += 'const vvl::unordered_map<std::string, std::string> api_extension_map {\n'
+        ext_fcn += 'const vvl::unordered_map<std::string, small_vector<std::string, 4>> api_extension_map {\n'
```

Comment:
```suggestion
        ext_fcn += 'const vvl::unordered_map<std::string, small_vector<std::string, 2>> api_extension_map {\n'
```
Suggesting 2 because right now I see at most 2 elements in the list

Reply:
Yes, I believe there are only 2. The reason I went with 4 is an attempt to "future proof" this a bit, but maybe that's not having the intended affect.

Reply:
Actually, I'll just grab the max list size during codegen; that's probably the "right" thing to do here.

Reply:
Is max size for all entries? In that case might look less pretty if we have outliers, so extra space should be paid by each entry. Not that we lose a lot of memory but if we expect that reasonable threshold (2-4) will work for most entries, then few allocations might be not an issue. Tradeoffs everywhere!